apps/emails/management/commands/email.py

```
@@ -0,0 +1,407 @@
+            fail_silently=False,
+        )
+        self.stdout.write(self.style.SUCCESS(f"Sent email to {', '.join(recipients)}"))
+
+    def _search_email(self, options: dict[str, Any]) -> None:
+        """Search inbound messages and print results as JSON."""
+
+        inbox_id = options.get("inbox")
+        if inbox_id is None:
+            inbox = EmailInbox.objects.filter(is_enabled=True).order_by("-priority", "id").first()
+            if inbox is None:
+                raise CommandError("No enabled inbox is available for search.")
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)